include/SDL3/SDL_render.h

```diff
@@ -2519,6 +2521,13 @@ extern SDL_DECLSPEC bool SDLCALL SDL_GetRenderVSync(SDL_Renderer *renderer, int
  *
  * The text is drawn in the color specified by SDL_SetRenderDrawColor().
  *
+ * **WARNING**: Note that in SDL 3.1.6, this function accepted a single string
```

Comment:
I would remove this warning. We can put it in the patch notes, but we don't need to carry it forward.

Reply:
Okay, that's fine.